values.yaml

```
@@ -345,6 +345,36 @@ server:
     # Manages license autoload. Required in Consul 1.10.0+, 1.9.7+ and 1.8.12+.
     enableLicenseAutoload: true
 
+  # A Kubernetes secret containing a certificate & key for the server agents to use
+  # for TLS communication within the Consul cluster. Cert needs to be carefully
+  # formed to work as if tls-init created it (reproducing what consul-k8s runs):
```

nit: Rephrasing this a bit
```suggestion
  # for TLS communication within the Consul cluster. Cert needs to be provided with
  # additional DNS name SANs so that it will work within the Kubernetes cluster:
```